crates/gpui/src/platform/windows/platform.rs

```diff
@@ -53,6 +57,7 @@ pub(crate) struct WindowsPlatform {
     clipboard_hash_format: u32,
     clipboard_metadata_format: u32,
     windows_version: WindowsVersion,
+    bitmap_factory: ManuallyDrop<IWICImagingFactory>,
```

Comment:
The `AgileReference<IWICImagingFactory>` is a reference to `IWICImagingFactory`, so we need to store it here to prevent any dangling pointer issues.